src/Networking/v2/Models/Subnet.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace OpenStack\Networking\v2\Models;
+
+use OpenStack\Common\Resource\AbstractResource;
+use OpenStack\Common\Resource\Listable;
+use OpenStack\Common\Resource\Creatable;
+use OpenStack\Common\Resource\Deletable;
+use OpenStack\Common\Resource\Retrievable;
+
+/**
+ * Represents a Networking v2 Network.
+ *
+ * @property \OpenStack\Networking\v2\Api $api
+ */
+class Subnet extends AbstractResource implements Listable, Retrievable, Creatable, Deletable
+{
+    public $id;
+    public $name;
+    public $enableDhcp;
+    public $networkId;
+    public $dnsNameservers;
+    public $allocationPools;
+    public $hostRoutes;
+    public $ipVersion;
+    public $gatewayIp;
+    public $cidr;
+    public $tenantId;
+    public $links;
+
+    protected $aliases = [
+        'enable_dhcp' => 'enableDhcp',
+        'network_id' => 'networkId',
+        'dns_nameservers' => 'dnsNameservers',
+        'allocation_pools' => 'allocationPools',
+        'host_routes' => 'hostRoutes',
+        'ip_version' => 'ipVersion',
+        'gateway_ip' => 'gatewayIp',
+        'tenant_id' => 'tenantId'
+    ];
+
+    protected $resourceKey = 'subnet';
+    protected $resourcesKey = 'subnets';
+
+    /**
+     * {@inheritDoc}
+     */
+    public function retrieve()
+    {
+        $response = $this->execute($this->api->getSubnet(), ['id' => (string) $this->id]);
+        $this->populateFromResponse($response);
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
we don't have a `bulkCreate` in `AbstractResource`, so we should define a brief description of what this operation does and its return type (`[]Subnet`)